source/_integrations/deconz.markdown

```diff
@@ -163,23 +170,7 @@ Navigate to **Developer tools->Events**. In the section **Listen to events** add
 
 To simplify using remote control devices in automations deCONZ integration exposes them as device triggers. This will expose all possible variations of button presses and rotations. Note that this is a manually curated list and will not initially be as complete as what deCONZ supports.
 
-Currently supported devices as device triggers:
-
-- Hue Dimmer Switch
-- Hue Smart Button
-- Hue Tap Switch
-- Friends of Hue Switch
-- Symfonisk Sound Controller
-- Trådfri On/Off Switch
-- Trådfri Open/Close Remote
-- Trådfri Remote Control
-- Trådfri Wireless Dimmer
-- Aqara Double Wall Switch
-- Aqara Mini Switch
-- Aqara Round Switch
-- Aqara Square Switch
-- Aqara Magic Cube
-- Aqara Opple 2/4/6 button Switches
+Review [this list](https://github.com/home-assistant/core/blob/15a8f6741bdf852b6f542be61aacb4896cd9e1ff/homeassistant/components/deconz/device_trigger.py#L559) to see if your device is supported by device triggers
```

Comment:
Ouch, I think we should not link end-user into code to figure something out

Reply:
I'll simplify